staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/time.go

```diff
@@ -55,6 +55,15 @@ func Now() Time {
 	return Time{time.Now()}
 }
 
+// String returns "<nil>" if the value is nil otherwise calls String on the
+// underlying time.Time
+func (t *Time) String() string {
+	if t == nil {
+		return "<nil>"
+	}
+	return t.Time.String()
+}
```

Comment:
Could you add a test for this as well [here](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/time_test.go)?

Reply:
Done. Please let me know if the there are other tests that should be added.